game_patch/misc/vpackfile.cpp

```diff
@@ -187,8 +231,13 @@ static int vpackfile_add_new(const char* filename, const char* dir)
     packfile->path[sizeof(packfile->path) - 1] = '\0';
     packfile->field_a0 = 0;
     packfile->num_files = 0;
-    // this is set to true for user_maps
-    packfile->is_user_maps = rf::vpackfile_loading_user_maps;
+    // packfile->is_user_maps = rf::vpackfile_loading_user_maps; // this is set to true for user_maps
+    // check for is_user_maps based on dir (like is_client_mods) instead of 0x01BDB21C
+    packfile->is_user_maps = (dir && (stricmp(dir, "user_maps\\multi\\") == 0 || stricmp(dir, "user_maps\\single\\") == 0));
```

Comment:
you should prepend std functions with `std::`. Otherwise if you don't depend on `string.h` instead of `cstring` and not use `use namespace std` it is undefined behavior whether those functions will be available in global namespace